assets/src/components/propertiesPanel/crowdingDiagram.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import ReactTooltip from "react-tooltip"
+import { crowdingIcon } from "../../helpers/icon"
+import { Crowding, OccupancyStatus } from "../../realtime"
+
+const extendedOccupancyStatus = (crowding: Crowding): OccupancyStatus => {
```

Comment:
Should this transformation be handled instead on the backend when we're parsing the busloc data? (If so, we could make `occupancyStatus` nonnullable.)